src/Cli/dotnet/commands/dotnet-test/IPC/Serializers/UnknownMessageSerializer.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.DotNet.Tools.Test;
+
+internal sealed class UnknownMessageSerializer : BaseSerializer, INamedPipeSerializer
+{
+    public int Id => 100;
```

Comment:
I would use int.MixValue so we go negative I don't think we will never use negative numbers to versioning the payloads.